src/familybot/lib/plugin_admin_actions.py

```
@@ -95,19 +95,20 @@ async def _rate_limit_full_scan() -> None:
-    except requests.exceptions.RequestException as e:
-        logger.error(f"Request error for {api_name}: {e}. URL: {response.request.url}")
+    except aiohttp.ClientResponseError as e:
+        logger.error(f"Request error for {api_name}: {e}. URL: {e.request_info.url}")
+    except aiohttp.ClientError as e:
+        logger.error(f"Client connection error for {api_name}: {e}")
     except json.JSONDecodeError as e:
-        logger.error(
-            f"JSON decode error for {api_name}: {e}. Raw: {response.text[:200]}"
-        )
+        text = await response.text()
+        logger.error(f"JSON decode error for {api_name}: {e}. Raw: {text[:200]}")
```

Ruff (0.15.1)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)